documentation/src/docs/segment-store-service.md

```diff
@@ -18,75 +18,76 @@ The Pravega Segment Store Service can provide the following guarantees:
-- _Segment Container Manager_: A component that can be used to determine the lifecycle of Segment Containers on a Pravega Node. This is used to start or stop Segment Containers based on an external coordination service (such as the Pravega Controller).
+- **Pravega Node**: A host running a Pravega Process.
+- **Stream Segment Container** (or **Segment Container**): A logical grouping of Stream Segments. The mapping of Segments to Containers is deterministic and does not require any persistent store; Segments are mapped to Containers via a hash function (based on the Segment's name).
+- **Durable Data Log Adapter** (or **Durable Data Log**): An abstraction layer for Tier 1 Storage.
+- **Storage Adapter**: An abstraction layer for Tier 2 Storage.
+- **Cache**: An abstraction layer for append data caching.
+- **Streaming Client**: An API that can be used to communicate with the Pravega Segment Store.
+- **Segment Container Manager**: A component that can be used to determine the lifecycle of Segment Containers on a Pravega Node. This is used to start or stop Segment Containers based on an external coordination service (such as the Pravega Controller).
 
 The Segment Store handles writes by first writing them to a log (_Durable Data Log_) on a fast storage (SSDs preferably) and immediately acking back to the client after they have been persisted there. Subsequently, those writes are then aggregated into larger chunks and written in the background to Tier 2 storage. Data for appends that have been acknowledged (and are in Tier 1) but not yet in Tier 2 is stored in the Cache (in addition to Tier 1). Once such data has been written to Tier 2 Storage, it may or may not be kept in the Cache, depending on a number of factors, such as Cache utilization/pressure and access patterns.
 
-More details about each component described above can be found in the [Components](#Components) section.
+More details about each component described above can be found in the [Components](#components) section.
 
 ## System Diagram
 
 ![System Diagram](img/Segment.Store.Components.png)
 
-In the above diagram, the major components of the Segment Store are shown. But for simplicity, only one Segment Container is depicted. All Container components and major links between them (how they interact with each other) are shown. The _Container Metadata_ component is not shown, because every other component communicates with it in one form or another, and adding it would only clutter the diagram.
+In the above diagram, the major components of the Segment Store are shown. But for simplicity, only one Segment Container is depicted. All Container components and major links between them (how they interact with each other) are shown. The _Container Metadata_ component is not shown, because every other component communicates with it in, one form or another, and adding it would only clutter the diagram.
```

Comment:
remove the added comma between `in` and `one`

Reply:
Accepted Suggestion